rasa/core/channels/mattermost.py

```diff
@@ -86,16 +156,27 @@ def blueprint(self, on_new_message):
         @mattermost_webhook.route("/webhook", methods=["POST"])
         async def webhook(request: Request):
             output = request.json
-            if output:
+
+            if not output:
+                return response.text("")
+
+            sender_id = output["user_id"]
+            self.bot_channel = output["channel_id"]
+                
+            # handle normal message with trigger_word
+            if "trigger_word" in output:
```

Comment:
To reduce the complexity of the method, you could put the code of the if block into a separate method as well as the code from the else block.

Reply:
Ok, done as well. 
I additionally added the possibility to send images as attachments.

Code is formatted with black.